custom/icds_reports/reports/fact_sheets.py

```diff
@@ -497,41 +497,72 @@ def new_table_config(self):
 
-                            },
-                            {
-                                'data_source': 'AggAWCMonthlyDataSource',
-                                'header': 'Adolescent girls (11-14 years)',
-                                'slug': 'cases_person_adolescent_girls_11_14_all',
-                                'average': [],
-
-                            },
-                            {
-                                'data_source': 'AggAWCMonthlyDataSource',
-                                'header': 'Adolescent girls (15-18 years)',
-                                'slug': 'cases_person_adolescent_girls_15_18_all',
-                                'average': [],
-
-                            },
-                            {
-                                'data_source': 'AggAWCMonthlyDataSource',
-                                'header': 'Adolescent girls (11-14 years)  enrolled for Anganwadi Services',
-                                'slug': 'cases_person_adolescent_girls_11_14',
-                                'average': [],
-
-                            },
-                            {
-                                'data_source': 'AggAWCMonthlyDataSource',
-                                'header': 'Adolescent girls (15-18 years)  enrolled for Anganwadi Services',
-                                'slug': 'cases_person_adolescent_girls_15_18',
-                                'average': [],
-
                             }
                         ]
                     },
                 ]
             }
         ]
 
+        if not self.beta:
+            fact_sheet_data_config[4]['sections'][0]['rows_config'].extend([
+                {
+                    'data_source': 'AggAWCMonthlyDataSource',
+                    'header': 'Adolescent girls (11-14 years)',
+                    'slug': 'cases_person_adolescent_girls_11_14_all',
+                    'average': [],
+
+                },
+                {
+                    'data_source': 'AggAWCMonthlyDataSource',
+                    'header': 'Adolescent girls (15-18 years)',
+                    'slug': 'cases_person_adolescent_girls_15_18_all',
+                    'average': [],
+
+                },
+                {
+                    'data_source': 'AggAWCMonthlyDataSource',
+                    'header': 'Adolescent girls (11-14 years)  enrolled for Anganwadi Services',
+                    'slug': 'cases_person_adolescent_girls_11_14',
+                    'average': [],
+
+                },
+                {
+                    'data_source': 'AggAWCMonthlyDataSource',
+                    'header': 'Adolescent girls (15-18 years)  enrolled for Anganwadi Services',
+                    'slug': 'cases_person_adolescent_girls_15_18',
+                    'average': [],
+
+                }
+            ]
+            )
+        elif self.config['month'] >= ADOLESCENT_GIRLS_DATA_THRESHOLD:
```

Comment:
is there a possibility for self.beta and self.config['month'] < ADOLESCENT_GIRLS_DATA_THRESHOLD?

Reply:
yeah, Thats a possibility in which case we don't want to show these indicators.